src/current/_includes/releases/v24.2/v24.2.0.md

```diff
@@ -0,0 +1,108 @@
+
+{% include releases/new-release-downloads-docker-image.md release=include.release advisory_key="a103220"%}
+
+<h3 id="v24-2-0-feature-highlights">Feature highlights</h3>
+
+This section summarizes the most significant user-facing changes in v24.2.0 and other features recently made available to CockroachDB users across versions. For a complete list of features and changes in v24.2, including bug fixes and performance improvements, refer to the [release notes]({% link releases/index.md %}#testing-releases) for previous v24.2 testing releases. You can also search the docs for sections labeled [New in v24.2](https://www.cockroachlabs.com/docs/search?query=new+in+v24.2).
+
+- **Feature categories**
+    - [Performance](#v24-2-0-performance)
+    - [Change data capture](#v24-2-0-change-data-capture)
+    - [Recovery](#v24-2-0-recovery)
+    - [Security](#v24-2-0-security)
+    - [Migrations](#v24-2-0-migrations)
+    - [SQL](#v24-2-0-sql)
+    - [Operations](#v24-2-0-operations)
+    - [Observability](#v24-2-0-observability)
+- **Additional information**
+    - [Backward-incompatible changes](#v24-2-0-backward-incompatible-changes)
+    - [Deprecations](#v24-2-0-deprecations)
+    - [Known limitations](#v24-2-0-known-limitations)
+    - [Additional resources](#v24-2-0-additional-resources)
+
+{{ site.data.alerts.callout_info }}
+In CockroachDB Self-Hosted, all available features are free to use unless their description specifies that an Enterprise license is required. For more information, refer to the [Licensing FAQ](https://www.cockroachlabs.com/docs/stable/licensing-faqs).
+{{ site.data.alerts.end }}
+
+<div id="feature-highlights">
+
+<h4 id="v24-2-0-performance">Performance</h4>
+
+{% include releases/v24.2/feature-highlights-performance.html %}
+
+<h4 id="v24-2-0-change-data-capture">Change Data Capture</h4>
+
+{% include releases/v24.2/feature-highlights-change-data-capture.html %}
+
+<h4 id="v24-2-0-recovery">Recovery</h4>
+
+{% include releases/v24.2/feature-highlights-recovery.html %}
+
+<h4 id="v24-2-0-security">Security</h4>
+
+{% include releases/v24.2/feature-highlights-security.html %}
+
+<h4 id="v24-2-0-migrations">Migrations</h4>
+
+{% include releases/v24.2/feature-highlights-migrations.html %}
+
+<h4 id="v24-2-0-sql">SQL</h4>
+
+{% include releases/v24.2/feature-highlights-sql.html %}
+
+<h4 id="v24-2-0-operations">Operations</h4>
+
+{% include releases/v24.2/feature-highlights-operations.html %}
+
+<h4 id="v24-2-0-observability">Observability</h4>
+
+{% include releases/v24.2/feature-highlights-observability.html %}
+
+<h4 id="v24-2-0-backward-incompatible-changes">Backward-incompatible changes</h4>
+
+Before [upgrading to CockroachDB v24.2]({% link v24.2/upgrade-cockroach-version.md %}), be sure to review the following backward-incompatible changes, as well as [key cluster setting changes](#v24-2-0-cluster-settings), and adjust your deployment as necessary.
+
+- [XXX](): ADD STUFF HERE
```

Comment:
```suggestion
CockroachDB v24.2 includes no backward-incompatible changes or deprecations.
```

Reply:
thanks Matt - made this change in my local copy which I'll push here shortly!